apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx

```diff
@@ -226,23 +226,30 @@ export const SignOnMethods: FunctionComponent<SignOnMethodsPropsInterface> = (
                                    gitHubAuthenticators: GenericAuthenticatorInterface[],
                                    facebookAuthenticators: GenericAuthenticatorInterface[]): void => {
 
-        eventPublisher.publish("application-sign-in-method-click-add", {
-            "method": loginFlow
-        });
-
         if (!loginFlow) {
             setModeratedAuthenticationSequence(authenticationSequence);
         } else if (loginFlow === LoginFlowTypes.DEFAULT) {
+            eventPublisher.publish("application-sign-in-method-click-add", {
+                "type": "default"
```

Comment:
`type` doesn't need to be enclosed within quotation marks IMO. Check other places as well.

Reply:
Fixed in #2729